src/lib/forms/helpers.ts

```
@@ -531,3 +531,420 @@
+		title_similarity: data.ranking.options.title_similarity,
+		remove_all_trash: data.ranking.options.remove_all_trash,
+		remove_ranks_under: data.ranking.options.remove_ranks_under,
+		remove_unknown_languages: data.ranking.options.remove_unknown_languages,
+		allow_english_in_languages: data.ranking.options.allow_english_in_languages,
+		languages_required: data.ranking.languages.required,
+		languages_exclude: data.ranking.languages.exclude,
+		languages_preferred: data.ranking.languages.preferred,
+		// Quality
+		...Object.entries(data.ranking.custom_ranks.quality).reduce(
+			(acc, [key, value]: [string, CustomRank]) => ({
+				...acc,
```

Biome

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)